src/taxes/components/TaxChannelsMenu/TaxChannelsMenu.tsx

```diff
@@ -0,0 +1,71 @@
+import { Card } from "@material-ui/core";
+import { Skeleton } from "@material-ui/lab";
```

Comment:
We have our own implementation 

```suggestion
import Skeleton from "@saleor/components/Skeleton";
```